src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -73,16 +74,8 @@ private void populatePathMappings(List<MappingConfig> mappingConfiguration) thro
         for (MappingConfig conf : mappingConfiguration) {
             String fromScheme = conf.getFromScheme();
             int priority = conf.getPriority();
-            Map<Integer, List<MappingConfig>> priority2ConfigPair = scheme2priority2ConfigPair.get(fromScheme);
-            if (priority2ConfigPair == null) {
-                priority2ConfigPair = new HashMap();
-                scheme2priority2ConfigPair.put(fromScheme, priority2ConfigPair);
-            }
-            List<MappingConfig> configPair = priority2ConfigPair.get(priority);
-            if (configPair == null) {
-                configPair = new ArrayList<>();
-                priority2ConfigPair.put(priority, configPair);
-            }
+            Map<Integer, List<MappingConfig>> priority2ConfigPair = scheme2priority2ConfigPair.computeIfAbsent(fromScheme, k -> new HashMap<>());
+            List<MappingConfig> configPair = priority2ConfigPair.computeIfAbsent(priority, k -> new ArrayList<>());
```

Comment:
I don't understand.  `priority2ConfigPair` is a Map; how is this different from [`Map#computeIfAbsent`](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-), introduced in Java **8**?

In general, this is a point _against_ using all-in-one packages such as Lombok: they tend to stick around long after the reasons for using them are gone.

Reply:
I don't really understand... 
This **is** [Map#computeIfAbsent](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-).
I used the functional style that Java 8 offers instead of the "if" statements from before.
 It has nothing to do with Lombok.